Tools/LocomotionTool/LocomotionTool.cs

```diff
@@ -228,7 +227,7 @@ void OnDestroy()
             this.RemoveRayVisibilitySettings(rayOrigin, this);
             this.ClearFeedbackRequests();
 
-            if (m_ViewerScaleVisuals)
+            if (m_ViewerScaleVisuals && m_ViewerScaleVisuals.gameObject.activeInHierarchy)
```

Comment:
@amirebrahimi do you remember why this changed?

Reply:
No idea at this point.

Reply:
OK. I will try reverting it and see what happens.

Reply:
I couldn't reproduce any issue around the viewer visuals in edit mode, play mode, or in a player build after reverting this change